news/2024/2024-07-28-the-perennial-concludes.md

```diff
@@ -0,0 +1,420 @@
+| Nipis Madu vs. AHHHHHHHHHHHH | Quarter Finals | [VOD](https://youtu.be/xpR6bljg-i8?si=QR0n2L4C-zE3Eb8h) |
+| Bob vs. european goats | Quarter Finals | [VOD](https://youtu.be/tL1zR-bQqrM?si=FjzRmtxdNRKQwsZe) |
+| the freak squad vs. Hatsune Miku | Quarter Finals | [VOD](https://youtu.be/nQT0GVh9POk?si=g-QIV-WctbhtN0nd) |
+| widepeepoHappy vs. Tea Party | Semi Finals | [VOD](https://youtu.be/zHCkywcpUBs?si=o22FJLSbaf7SMdV9) |
+| together we are terrific vs. Hatsune Miku | Semi Finals | [VOD](https://youtu.be/3JwWWOKOhc8?si=8aSoB71_BGuCDNeN) |
+| mom's touch vs. EMPICONES | Finals | [VOD](https://youtu.be/5HcQRVBM8Iw?si=RGFIwtpbefvu1RU_) |
+| APAC RADIANT vs. together we are terrific | Grand Finals | [VOD](https://youtu.be/XRG7ewQExew?si=axH3ios8RxmJf62l) |
+
+### Highlight videos
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/AJEq85TldL4?si=3RTM2LkE7U-5Jrcr" frameborder="0" allowfullscreen></iframe>
+</div>
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/iNx0xQtboq0?si=WmxSn-rtJcydRiVB" frameborder="0" allowfullscreen></iframe>
+</div>
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/eSASm3RZpOk?si=mXioukIsmw-r4kjp" frameborder="0" allowfullscreen></iframe>
+</div>
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/GHcFCjpFKlE?si=5mTY5j3Bz2jNN_Et" frameborder="0" allowfullscreen></iframe>
+</div>
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/C-8bLhId9yY?si=QRICzvpxF8p_vpi-" frameborder="0" allowfullscreen></iframe>
+</div>
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/kx4HYPgFIFs?si=STcQRT5IzooRV4mo" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Original songs
+
+<style>
+    .osu-md--news .audio-player {
+        max-width: inherit;
+    }
+</style>
+
+The Perennial stepped it up this year with six original songs: one for each round of the tournament, plus a second original in grand finals. Each of them had their own original artwork, and the majority had video introductions. Of course, the tournament’s final boss, the grand finals tiebreaker, had all that and a full storyboard. Check out all six songs, as with a variety of genres and styles, there’s sure to be something for everyone. If you are interested in the video intros and the maps for all these originals, those can be found on our [YouTube](https://youtube.com/c/ThePerennialYT).
+
+### Group Stage
+
+[![](https://assets.ppy.sh/beatmaps/2198890/covers/raw.jpg)](https://osu.ppy.sh/beatmapsets/2198890#osu/4653331)
```

Comment:
these images are fairly tall... oh well, maybe not too many yet

upload these to the news folder on the repository instead of linking the beatmap cover image